testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -220,6 +219,16 @@ def test_load_new(self):
         u.load_new(PDB_small)
         assert_equal(len(u.trajectory), 1, "Failed to load_new(PDB)")
 
+    def test_load_new_returns_Universe(self):
+        u = mda.Universe(PSF)
+        result = u.load_new(PDB_small)
+        assert result is u
+
+    def test_load_new_None_returns_Universe(self):
+        u = mda.Universe(PSF)
+        result = u.load_new(None)
```

Comment:
See comment: It allows building a universe *without coordinates*.

Otherwise `Universe(PSF)` fails. We could probably change this behaviour at the `Universe` level but right now it is *required*. (Try for yourself...)